cmd/flags.go

```diff
@@ -328,6 +351,21 @@ func ValidateNoArgs(ctx *cli.Context) error {
 	return nil
 }
 
+func ValidateNetworkFlags(ctx *cli.Context) error {
```

Comment:
If you move this function into `prysm/config/features`, then you don't need to duplicate `SepoliaTestnet`, `HoleskyTestnet`, `Mainnet` and `NetworkFlags`.

Reply:
`ValidateNetworkFlags` is a public function. Please as a documentation
```
// ValidateNetworkFlags ...
func ValidateNetworkFlags(...
```

Reply:
resolved this in the next commit